test/nightwatch/table/table-test.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  '@tags': ['custom', 'table'],
+  'disabled': true,
```

Comment:
Yes, and also because I was trying to figure out how to test each cell in a table. Hopefully will get this sorted out before this PR gets merged.